vertx-mongo-client/src/main/java/examples/Examples.java

```diff
@@ -547,4 +553,268 @@ public void example16_d1(MongoClient mongoClient) throws Exception{
 
     });
   }
+  public void example17(MongoClient mongoClient) {
+    mongoClient.createGridFsBucketService("fs", res -> {
+      if (res.succeeded()) {
+        MongoGridFsClient client = res.result();
+        //Interact with the GridFS client...
+      } else {
```

Comment:
redundant comment?  Or at least move it above the line that it refers to - this goes for the following samples as well